### ISSUE TITLE
Supplemental Guidance (for what isn't in WCAG 2 and what probably should be in WCAG 3) https://www.w3.org/WAI/GL/low-vision-a11y-tf/wiki/Supplemental_Guidance_List

### ISSUE THREAD

**Issue by ChrisLoiselle** [w3c/low-vision-a11y-tf#92]
Supplemental Guidance (for what isn't in WCAG 2 and what probably should be in WCAG 3) https://www.w3.org/WAI/GL/low-vision-a11y-tf/wiki/Supplemental_Guidance_List
